storage-s3-dynamodb/integration_tests/dynamodb_logstore.py

```diff
@@ -85,20 +86,26 @@
     .appName("utilities") \
     .master("local[*]") \
     .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
-    .config("spark.delta.logStore.class", delta_storage) \
-    .config("io.delta.storage.ddb.tableName", dynamo_table_name) \
-    .config("io.delta.storage.ddb.region", dynamo_region) \
-    .config("io.delta.storage.errorRates", dynamo_error_rates) \
+    .config("spark.delta.logStore.s3.impl", delta_storage) \
+    .config("spark.delta.logStore.s3a.impl", delta_storage) \
+    .config("spark.delta.logStore.s3n.impl", delta_storage) \
```

Comment:
let's idiot-proof this to work with s3, s3a, s3n

Reply:
+1